debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCClassArrayClassSlotIterator.java

```diff
@@ -47,6 +50,13 @@ protected GCClassArrayClassSlotIterator(J9ClassPointer clazz) throws CorruptData
 			slots.add(slot);
 			addresses.add(VoidPointer.cast(clazz.arrayClassEA()));
 		}
+		if (valueTypeHelper.isJ9ClassAValueType(clazz)) {
+			slot = clazz.nullRestrictedArrayClass();
+			if(slot.notNull()) {
+				slots.add(slot);
+				addresses.add(VoidPointer.cast(clazz.nullRestrictedArrayClassEA()));
+			}
+		}
 		if(clazz.romClass().modifiers().allBitsIn(J9AccClassArray)) {
```

Comment:
While you're here, can you please fix the missing space before `(` on this line as well?